src/main/java/com/moodybluez/enterprise/MoodyBluezController.java

```diff
@@ -35,16 +40,26 @@ public String showRegistrationForm(Model model) {
         return "index";
     }
 
+    @GetMapping("/registration")
+    public String showRegistration(User user) {
+
+        return "registration";
+    }
+
+
 
     @PostMapping("/process_register")
-    public String processRegister(User user) throws Exception {
+    public String processRegister(User user){
         BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
         String encodedPassword = passwordEncoder.encode(user.getPassword());
         user.setPassword(encodedPassword);
 
-        userService.save(user);
-
+        try {
+            userService.save(user);
+            user.setPassword(" ");
+        } catch (Exception e) {
+            return "registration_error";
```

Comment:
Ok, good.  Show the user an error if something went wrong.

Also, use proper logging.  Add a log statement in a catch block to log to error priority.